### DISCUSSION TITLE
Sensitive landmine.

### DISCUSSION THREAD

**Discussion by yujispin990** [winstxnhdw/lc-hax#86] [Ideas]
a ripped idea from another project, making landmine explode on the when a person enters the explosion radius of the landmine. But maybe an option to only target teammates and not us? of course again like before, it is totally understandable if you were to reject this idea.

**Comment by yujispin990** [comment]
Thank you <3

**Reply by winstxnhdw** [reply]
I'll think about whether it's worth to implement this. I can imagine the performance cost of this in large lobbies. Probably has to be toggleable.
